src/drive/web/modules/views/Upload/useResumeFromFlagship.ts

```diff
@@ -46,4 +54,24 @@ export const useResumeUploadFromFlagship = (): void => {
 
     void doResumeCheck()
   }, [client, dispatch, webviewIntent, uploadQueue])
+
+  useEffect(() => {
+    // The upload can only happen once as the webapp is closed before the upload process can start again
+    if (didFlow.current) return
+
+    const processed = (getProcessed(state) as () => unknown[]).length
```

Comment:
This typing seems weird. Shouldn't this be `getProcesses(state) as unknown[]`? Otherwise you seems to call `.length` on a method pointer.

Reply:
that's because I'm casting the actual getProcessed selector instead of the result, in any case it doesn't look great when using untyped code in strictly typed code